cli-tests/src/helper.ts

```diff
@@ -24,6 +52,20 @@ export const isFileEmpty = (file: string): boolean  => {
     } 
 };
 
+export const createFiles = (absolutePath: string, files: string[]) => {
```

Comment:
Just wondering why you don't use Typescript itself to create files? It would be platform-independent.

Reply:
Good idea, I will change it

Reply:
done